app/views/idv/cancellations/destroy.html.slim

```diff
@@ -0,0 +1,7 @@
+= render 'shared/failure', presenter: @presenter
+
+ul class="list-reset #{@presenter.state_color}-dots"
+  - @presenter.cancellation_effects.each do |effect|
+    li = effect
+
+.mt2 = link_to t('idv.cancel.return_to_account'), account_path
```

Comment:
Yeah, lemme try the `link_to :back` and see what I can figure out.

Also, on the link to the sp return url, we decided to handle that in a separate follow-on PR since this one is already pretty hefty. There's a separate ticket for it this sprint.